Valigator.Core/Core/Model.cs

```diff
@@ -126,17 +148,20 @@ private static (PropertyInfo[] dataProperties, MemberInfo[] validateContentsMemb
 				.Where(p => p.GetCustomAttribute<ValidateContentsAttribute>() != null)
 				.ToArray();
 
-			var noGetters = dataProperties
-				.Concat(validateContentsProperties)
-				.Where(x => x.GetMethod == null)
-				.ToArray();
+			if (!(model is ValigatorModelBase))
```

Comment:
If it's a ValigatorModel, we fake the getter and setter, so we don't really care if there is one on the property.